lib/ts/recipe/webauthn/recipeImplementation.ts

```diff
@@ -354,9 +354,29 @@ export default function getRecipeImplementation(
                 fetchResponse,
             };
         },
-        registerAndSignUp: async function ({ email, options, userContext }) {
+        registerUser: async function ({ registrationOptions }) {
```

Comment:
i think this should contain some credential reference in the name, as the main purpose of this method is to generate the credential  for a user (in order for the credential to be registered - either on sign up or, in the future, on registerCredential in their "profile" )